boards/posix/native_posix/doc/index.rst

```diff
@@ -587,8 +587,6 @@ UART
 
 This driver can be configured with :option:`CONFIG_UART_NATIVE_POSIX`
 to instantiate up to two UARTs. By default only one UART is enabled.
-With :option:`CONFIG_UART_NATIVE_POSIX_PORT_1_ENABLE`
-you can enable the second one.
 
```

Comment:
As there is now no configure option to enable the second uart, do we not need to describe how to get the second uart enabled?
The native_posix.dts only defines one uart, so perhaps we would need to explain the process of hoe to get the second uart added?

Reply:
We can add the second serial port to the DTS and mark it 'disabled'.

Reply:
That is a possibility, but could we not leave the UART_NATIVE_POSIX_PORT_1_ENABLE Kconfig option to enable uart 1? Then we could also leave the depends on for the Kconfig option UART_NATIVE_WAIT_PTS_READY_ENABLE. Or perhaps it is not possible for a Kconfig option to set a DT status option to okay?

Reply:
Its not possible.  We parse DTS before Kconfig, so we can't use any Kconfig to set DTS info.